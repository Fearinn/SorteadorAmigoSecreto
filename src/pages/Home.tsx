import Card from "../components/Card";
import Formulario from "../components/Formulario";
import ListaDeParticipantes from "../components/ListaDeParticipantes";
import Rodape from "../components/Rodape";

const Home = () => {
  return (
    <>
      <Card>
        <Formulario />
        <ListaDeParticipantes />
        <Rodape />
      </Card>
    </>
  );
};

export default Home;
