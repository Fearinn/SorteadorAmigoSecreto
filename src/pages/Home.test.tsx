import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";
import Home from "./Home";

describe("página inicial", () => {
  test("a renderização ocorre corretamente", () => {
    const {container}  = render(
      <RecoilRoot>
        <Home />
      </RecoilRoot>
    );

    expect(container).toMatchSnapshot();
  });
});
