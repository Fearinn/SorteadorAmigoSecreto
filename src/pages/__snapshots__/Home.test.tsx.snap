// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`página inicial a renderização ocorre corretamente 1`] = `
<div>
  <div
    class="card"
  >
    <form
      class="formulario"
    >
      <h1
        class="titulo"
      >
        Vamos começar!
      </h1>
      <fieldset
        class="fieldset"
      >
        <div
          class="container"
        >
          <input
            class="input"
            placeholder="Insira os nomes dos participantes. Pelo menos 3 são necessários para brincar."
            required=""
            type="text"
            value=""
          />
          <button
            class="botao"
            disabled=""
            type="submit"
          >
            Adicionar
          </button>
        </div>
        <input
          class="input"
          placeholder="Defina uma palavra para identificar seu jogo. Ex.: familiasilva123"
          type="text"
          value=""
        />
      </fieldset>
      
    </form>
    <footer
      class="rodape"
    >
      <button
        class="botao"
        disabled=""
      >
        Iniciar brincadeira
      </button>
      <div
        class="imagem"
      />
    </footer>
  </div>
</div>
`;
